bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/structures/TapoBridgeConfiguration.java

```diff
@@ -68,7 +68,7 @@ public void loadSettings() {
             username = config.get(CONFIG_EMAIL).toString();
             password = config.get(CONFIG_PASS).toString();
             cloudDiscoveryEnabled = Boolean.parseBoolean(config.get(CONFIG_DISCOVERY_CLOUD).toString());
-            discoveryIntervalM = Integer.valueOf(config.get(CONFIG_DISCOVERY_INTERVAL).toString());
+            discoveryIntervalM = (int) Double.parseDouble(config.get(CONFIG_DISCOVERY_INTERVAL).toString());
```

Comment:
This work-around should not be needed in 3.4 as fixed by openhab/openhab-core#3024. See also #13112.